install/install.sh

```diff
@@ -50,14 +50,6 @@ verifySupported() {
         fi
     done
 
-    if [ "$current_osarch" == "darwin-arm64" ]; then
```

Comment:
So we need a bit more robust fallback than this. The install.sh script is unversioned and live, which means this change will break M1 macs until such time as the release is published. I think what we need to do is "if darwin-arm64 && darwin/arm64 exists and is public then download it else amd64"

Reply:
Makes sense -- keep in mind this script is local to the 1.5 branch right now, but I agree we should have this.

Reply:
> Makes sense -- keep in mind this script is local to the 1.5 branch right now, but I agree we should have this.

True, but this also means it won't work until we merge release branch back to master, and thus we'll have no way to test it works before we go live.